docs/prow/manage-component-jobs-with-templates.md

```diff
@@ -12,124 +12,164 @@ This document describes how to define, modify, and remove Prow jobs for Kyma com
+                 type_postsubmit: "true"
+                 cluster: "trusted-workload"
+              ...
+           jobConfigs:
+              - repoName: "kyma-project/kyma"
+                jobs:
+                   - jobConfig:
+                        name: "skr-aws-upgrade-integration-dev"
+                        cron: "0 */4 * * *" # "Every four hours"
+                        optional: true
+        ...
+   ```
 
-```yaml
-  - from: templates/generic-component.yaml
-    render:
-      - to: ../prow/jobs/kyma/components/application-broker/application-broker-generic.yaml
-        values:
-          <<: *kyma_generic_component
-          path: components/application-broker
-          since: "1.7"
-          optional: true
-    ...
-```
+   Such an entry uses the `generic.tmpl` template to create the `skr-aws-upgrade-integration-dev.yaml` file under
+the `/prow/jobs/kyma/` subfolder, specifying that postsubmit job for this test.
+Set the **optional** parameter to `true` for this job to be optional on pull requests (PRs), not to block others.
+**Cron** parameter indicates that this Prow job is run every four hours.
 
-Such an entry uses the `generic-component.yaml` template to create the `application-broker-generic.yaml` file under the `/prow/jobs/kyma/components/application-broker/` subfolder, specifying that the presubmit and postsubmit jobs for this component should apply from the `1.7` release onwards. Set the **optional** parameter to `true` for this job to be optional on pull requests (PRs), not to block others.
+   If needed, global config sets (**globalSets**) can be added to the `templates/config.yaml` file.
 
-> **NOTE:** Make sure that the `.yaml` file and the component folder name are the same as the name of the Kyma component. Also, all `.yaml` files in the whole `jobs` structure need to have unique names.
+   For **component job**, instead of `name`, `path` indicating the path to the component, is required.
+   For example:
 
-Use the buildpack for Go or Node.js applications provided in the `test-infra` repository. It is the standard mechanism for defining Prow jobs. If the buildpack you want to use is not there yet, you have to add it. When you add a new buildpack, follow the example of the already defined ones.
+   ```yaml
+   templates:
+     - from: templates/generic.tmpl
+       render:
+         - to: ../prow/jobs/control-plane/components/provisioner/provisioner-generic.yaml
+           jobConfigs:
+             - repoName: "github.com/kyma-project/control-plane"
+               jobs:
+                 - jobConfig:
+                     path: components/provisioner
+   ```
 
-2. Define a test for your component.
+   - For more information about creating template file, as well as local config sets (**localSets**), job configs (**jobConfig**) and
+     (**globalSets**), please refer to [specific documentation](https://github.com/kyma-project/test-infra/tree/main/development/tools/cmd/rendertemplates).
```

Comment:
```suggestion
```

Reply:
Should I also change `specific documentation` to `Render Templates` as it was in other cases?